assets/css/components/custom_modal.css

```diff
@@ -14,25 +14,53 @@
         bottom / cover url("/images/backgrounds/menu-bg.jpg")
 }
 
+.rounds {
```

Comment:
You should prefix this with the class of the modal so it doesn't bleed over into other styles. This applies to all the other selectors in the changes as well. It should be fine cause I don't think we're using these classes elsewhere, but it's better practice to be careful about this anyway especially if we haven't tested them thoroughly yet.